vendor/engines/bulk_email/app/mailers/bulk_email/mailer.rb

```diff
@@ -2,16 +2,24 @@ module BulkEmail
 
   class Mailer < BaseMailer
 
-    def send_mail(recipient:, custom_subject:, facility:, custom_message:)
+    def send_mail(recipient:, custom_subject:, facility:, custom_message:, subject_product_id: nil)
```

Comment:
As a general thing, do we care that the product is tied to the subject? Would `product_id` as a name suffice? Do we already have the product loaded when we call this method? If so, could we just pass `product`? Is this going through ActiveJob/GlobalID? If not, we should probably do `facility` as `facility_id`.